### ISSUE TITLE
File Explorer and Outline left padding are inconsistent 

### ISSUE THREAD

**Issue by rebornix** [microsoft/vscode#216539]
Does this issue occur when all extensions are disabled?: Yes/No

<img width="482" alt="image" src="https://github.com/user-attachments/assets/dcc9a769-6c6b-4b3f-9bcd-894e6fa9cc6e">

As shown above, the indentation in the outline is consistent while there are some additional left padding in the file explorer, pushing the tree view to the right side a bit more.

**Comment by benibenj** [comment]
I believe this is by design and only the case with the file explorer. @lramos15 probably knows more about this. What is the main reason for this?

**Comment by lramos15** [comment]
I'm not sure maybe @isidorn knows since he owned explorer before me.

I know that when we have a file icon theme that has folders the indentation can also change, so maybe that accounts for some of it.

**Comment by isidorn** [comment]
Yes, I think it was related to the file icon theme change. I know @aeschli was looking into the alignment with and without icons.

My knee jerk reaction - changing this is opening a can of worms.
